src/test/java/org/elasticsearch/percolator/PercolatorBackwardsCompatibilityTests.java

```diff
@@ -22,15 +22,18 @@
 import org.elasticsearch.action.index.IndexRequestBuilder;
 import org.elasticsearch.action.percolate.PercolateResponse;
 import org.elasticsearch.action.percolate.PercolateSourceBuilder;
+import org.elasticsearch.common.settings.ImmutableSettings;
```

Comment:
Not that the test has been moved to a different class, all the modifications to this file can be undone, so it doesn't appear in the list of changed files.